src/backend/api/Fusion.Resources.Api/Controllers/Requests/ApiModels/ApiProposedPerson.cs

```diff
@@ -18,13 +20,17 @@ public ApiProposedPerson(QueryResourceAllocationRequest.QueryProposedPerson prop
             {
                 ResourceOwner = new ApiPerson(proposedPerson.ResourceOwner);
             }
-        }
 
+            if (proposedPerson.DelegatedResourceOwners is not null)
+            {
+                DelegatedResourceOwners = new List<FusionPersonProfile?>(proposedPerson.DelegatedResourceOwners);
+            }
+        }
 
         public DateTimeOffset ProposedAt { get; set; }
         public ApiPerson Person { get; set; }
         public bool WasNotified { get; set; }
-
         public ApiPerson? ResourceOwner { get; set; }
+        public List<FusionPersonProfile?> DelegatedResourceOwners { get; set; }
```

Comment:
Incorrect use of model.
Should be something like: 
public IEnumerable<ApiPerson>? DelegatedResourceOwners { get; set; }